ui/src/components/Publication/Creation/StepFive.tsx

```diff
@@ -1,9 +1,93 @@
+        (state: Types.PublicationCreationStoreType) => state.conflictOfInterestStatus
+    );
+    const conflictOfInterestText = Stores.usePublicationCreationStore(
+        (state: Types.PublicationCreationStoreType) => state.conflictOfInterestText
+    );
+    const licence = Stores.usePublicationCreationStore((state: Types.PublicationCreationStoreType) => state.licence);
+    const content: string = Stores.usePublicationCreationStore(
+        (state: Types.PublicationCreationStoreType) => state.content
+    );
+
     return (
         <div className="text-sm text-grey-800 transition-colors duration-500 dark:text-white">
-            <p className="mb-2">Coming soon...</p>
-            <p className="mb-2">This will be a page to review all content before publishing.</p>
-            <p className="mb-2"> It is not possible to make any changes post-publication.</p>
+            <div className="relative mt-20 border-b border-grey-100 pb-8 transition-colors duration-500 dark:border-grey-700">
+                <span className="mb-4 block font-montserrat text-xl font-semibold text-pink-500">
+                    {Helpers.formatPublicationType(type)}
+                </span>
+                <h1 className="mb-4 block font-montserrat text-3xl text-grey-800 transition-colors duration-500 dark:text-white">
+                    {title}
+                </h1>
+                {title.length ? <Completed /> : <Incomplete />}
+            </div>
+            <div className="relative mt-10 border-b border-grey-100 pb-16 transition-colors duration-500 dark:border-grey-700">
+                <span className="mb-8 block text-xxs font-bold uppercase tracking-widest text-grey-800 transition-colors duration-500 dark:text-grey-100">
+                    Chosen publications to link
+                </span>
+                <p>TODO: Link information here</p>
+            </div>
+            <div className="relative mt-10 border-b border-grey-100 pb-16 transition-colors duration-500 dark:border-grey-700">
+                <span className="mb-8 block text-xxs font-bold uppercase tracking-widest text-grey-800 transition-colors duration-500 dark:text-grey-100">
+                    Creative commons licence
+                </span>
+                <Components.Link
+                    href={`${Config.values.licenceTypes.find((type) => type.value === licence)?.link}`}
+                    openNew={true}
+                    className="mb-2 block w-fit rounded underline decoration-teal-500 decoration-2 underline-offset-2 outline-0 hover:decoration-teal-600 focus:ring-2 focus:ring-yellow-400"
+                >
+                    <strong className="text-grey-800 transition-colors duration-500 dark:text-white">
+                        {Config.values.licenceTypes.find((type) => type.value === licence)?.nicename} 4.0
+                    </strong>
+                </Components.Link>
+                {licence ? <Completed /> : <Incomplete />}
```

Comment:
I'm gonna be super super picky here (sorry). Can we call it `CompletedIcon`? Just so it's a bit clearer when reading it from here?